bot/exts/filters/webhook_remover.py

```diff
@@ -72,7 +75,10 @@ async def on_message(self, msg: Message) -> None:
 
         matches = WEBHOOK_URL_RE.search(msg.content)
         if matches:
-            await self.delete_and_respond(msg, matches[1] + "xxx")
+            async with self.bot.http_session.delete(matches[0]) as resp:
```

Comment:
After testing, this still doesn't work. It seems the regex does not capture the `https://` part of the webhook, this will need to be added back in before you can make a request to delete it.

Reply:
That's strange, I've tested out the regEx and by using `matches[0]` it worked just fine, it included the whole url.

Reply:
Turns out the regex did not match the `https://` part in case of canary urls. The regEx pattern has to be changed.